### ISSUE TITLE
Doing nothing on Ubuntu?

### ISSUE THREAD

**Issue by Queuecumber** [Hackafe/atom-sync-settings#25]
Not sure what I'm doing wrong here. I have two ubuntu 14.04 setups. I installed and went through the steps on one machine, uploaded my settings just fine. I did the same thing on another, practically identical machine, and I was unable to download or upload. I even tried completely removing atom and my .atom folder and trying again. Nothing is printed in the debug log, no error messages appear anywhere. Any idea how I can troubleshoot this?

**Comment by RossBarnie** [comment]
Can you clone the gist via command line on the non-working machine? Have you made sure you have the same settings for the package on both machines?

**Comment by Queuecumber** [comment]
Yes I can clone the gist from the command line on the non-working machine, I tried with both machines having the same gist settings and auth token settings and I tried with each having their own auth token, neither configuration worked

**Comment by RossBarnie** [comment]
Hmm... Error reporting is probably one of the greatest weaknesses of this package at the moment unfortunately. If I were you I'd load the package for development, throw a bunch of `console.log()`s around and see what happens. Sorry I can't be more specific.

**Comment by jakubhruby** [comment]
I think this is not only on Ubuntu. I experience this on Win7, too. 
Last time I was able to sync was 2/9/2015 in the morning. Now I always get the "ok" notification when doing sync, but no revision is made on gist. 
I am able to clone repo and commit changes manually. I have also reinstalled the package and regenarated the token, but nothing helped. 
This is quite serious issue to me, because I use Atom on three devices and this feature is very important to me.

**Comment by jakubhruby** [comment]
Now it works, strange. A kind of magic, I suppose.

**Comment by jakubhruby** [comment]
Ok, this worked for me: I realised that i have installed the altgr package, which overrides every ctrl-alt- keybinding. Well, it overrides also events fired by sync-settings, which I have mapped to ctrl-alt-pageup/pagedown. However, despite nothing was synchronised, sync-settings showed me the "sync successfull" notifier, which was a little bit confusing.
After removing altgr package sync is working as expected.
